packages/protobuf/package.json

```diff
@@ -10,10 +10,9 @@
   },
   "scripts": {
     "clean": "rm -rf ./dist/*",
-    "build": "npm run build:cjs && npm run build:esm && npm run build:proxy",
+    "build": "npm run build:cjs && npm run build:esm",
```

Comment:
Can you also delete scripts/gen-esm-proxy.mjs ?

Reply:
Omg, I swear I removed this. I must not have pushed it.